jobma/permissions.py

```diff
@@ -0,0 +1,15 @@
+"""jobma permission classes"""
+from django.conf import settings
+
+from rest_framework.permissions import BasePermission
+
+
+class JobmaWebhookPermission(BasePermission):
+    """Restrict access to jobma via access token"""
+
+    def has_permission(self, request, view):
+        header = request.headers.get("authorization", "")
+        if header.startswith("Token "):
+            token = header[len("Token "):]
+            return token == settings.JOBMA_WEBHOOK_ACCESS_TOKEN
```

Comment:
I think this should return `False` and log an error if `JOBMA_WEBHOOK_ACCESS_TOKEN` isn't set as a fail-safe mode.